glusterd2/commands/volumes/volume-stop.go

```diff
@@ -143,6 +143,11 @@ func volumeStopHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if volinfo.State != volume.VolStarted {
```

Comment:
@Madhu-1  Can we remove  if (volinfo.State == volume.VolStopped) condition ?

Reply:
in both the conditions we are returning an appropriate error message, one is volume already in stopped state and another one is volume not started.

Reply:
It's required, if the volume is already stopped then no need of stopping it again

Reply:
@Madhu-1  Can't we have a single message saying "volume is not in started state". I don't see the need for an extra check.

Reply:
if we send only one error message for both the scenario it might be difficult for the user to identify what has happened.
Ex: currently in CSI driver if the volume is already in the stopped state it will proceed for volume deletion.
if we give single error message it might be difficult for the consumer to take appropriate action.